tests/testthat/test_atype.R

```diff
@@ -2,10 +2,11 @@ test_that( "works as expected", {
 
   t = data.frame(
     
-    date = c( '2014-01-01', '1/1/2014', '20140101' ),
-    character = c( 'A', 'B', 'C' ),
-    numeric = c( '-13', '20%', '$30.00' ),
-    logical = c( 'TRUE', 'FALSE', '0' ),
+    date = c( '2014-01-01', '1/1/2014', '20140101', NA ),
```

Comment:
Awesome, thanks for expanding this test to cover the new functionality! I don't think I asked you to do this, so this shows some out-of-the-box thinking - exactly the direction we want to be moving in.